doc/doxygen-root/README.md

```diff
@@ -0,0 +1,60 @@
+# CBMC documentation
+
+This is the root of CBMC documentation.
+
+## Install doxygen
+
+Building this documentation requires `doxygen` and `graphviz`.
+
+* On MacOS,
+  we recommend installing with [brew](https://brew.sh/):
+  ```
+  brew install doxygen graphviz
+  ```
+* On Ubuntu, we recommend installing with using `apt`:
+  ```
+  sudo apt install doxygen graphviz
+  ```
+
+Note: As of
+
+## Build the documentation
+
+To build the documentation, run
+
+```
+doxygen
+```
+
+The documentation takes about five minutes to build and is written to
+the `html` subdirectory as a set of 2600+ html files.  To view the
+documentation,
+open the file `html/index.html` in a web browser.  One MacOS, run
+
+```
+open html/index.html
+```
+
+## Publish the documentation
+
+To publish the documentation, we use the GitHub workflow
+[publish.yaml](../../.github/workflow/publish.yaml) that
+publishes the contents of the `html` subdirectory to
+the `github.io` web site.
+
+## Contribute documentation
+
+Coming soon...
```

Comment:
There appears to be some documentation on this at `contributing.md`.

Reply:
Added a link.  Thanks.